<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransmitterAdmin</class>
 <widget class="QDialog" name="TransmitterAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>6</y>
     <width>531</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="CreateTab">
       <attribute name="title">
        <string>Create</string>
       </attribute>
       <widget class="QToolBox" name="toolBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>501</width>
          <height>441</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="readFromFile">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>501</width>
           <height>379</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Read values from file</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>5</y>
            <width>491</width>
            <height>291</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="browseFile">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="fileTableWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="addAllBtn">
               <property name="text">
                <string>Add transmitters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearFileBtn">
               <property name="text">
                <string>Clear values</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>300</y>
            <width>491</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tstatusLabel1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="readFromManual">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>501</width>
           <height>379</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Enter values manually</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>5</y>
            <width>491</width>
            <height>311</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,6">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X Coordinate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="xInput"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y Coordinate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="yInput"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="addBtnManual">
               <property name="text">
                <string>Add transmitter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearBtnCreate">
               <property name="text">
                <string>Clear values</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>320</y>
            <width>491</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tstatusLabel2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="ModifyTab">
       <attribute name="title">
        <string>Modify</string>
       </attribute>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>80</y>
          <width>491</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>371</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Choose a transmitter to edit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tmitDropDown">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>491</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="modifyLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xInputModify"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Y Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yInputModify"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QPushButton" name="ModifyBtn">
             <property name="text">
              <string>Apply transmitter changes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>491</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tstatusLabel3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="RemoveTab">
       <attribute name="title">
        <string>Remove</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>401</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="removeBtn">
             <property name="text">
              <string>Remove Records</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clrSelectBtn">
             <property name="text">
              <string>Clear Selections</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tstatusLabel5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="ListTableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>501</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ListTab">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <widget class="QTableView" name="ListTableView">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>511</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tExportBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>450</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Export to File</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>450</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tstatusLabel4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="helpBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
